### PR TITLE
fix: Releases documentation tag view not fixed.

### LINKED ISSUE #1032: Release Notes should be optional
## Feature request
Release Notes should be optional and/or the tab should be removable.

#### Describe the solution you'd like
The Release Note are always present; it should be possible to either configure the system that hey aren't displayed or get rid of them like any other window.
When a user has many tabs open, it can be annoying having a tab there always taking some space.

Selection_018

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
#### Steps to reproduce

1. Login.
2. Open Releases Notes in sidebar menu.
3. Close Documentation tag view.

#### Screenshot or Gif

https://user-images.githubusercontent.com/20288327/128945133-f45e72a9-af9d-491f-b94f-518a64c265e8.mp4

#### Other relevant information
* Your OS: Linux Mint 19.1 Cinnamon x64. 
* Browser: Mozilla Firefox 88.0.1.
* Node.js version: 14.17.0.
* NPM version: 7.19.0.
* adempiere-vue version: 4.3.1.

#### Additional context
fixes https://github.com/adempiere/adempiere-vue/issues/1032

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -92,13 +92,21 @@ export const constantRoutes = [
...
+      icon: 'documentation',
+      breadcrumb: false
+    },
     children: [
       {
         path: 'index',
         component: () => import('@/views/documentation/index'),
         name: 'Documentation',
-        meta: { title: language.t('documentation.releaseNotes'), icon: 'documentation', affix: true, isIndex: true }
+        meta: {
```
Comment (yamelsenih) [inline src/router/index.js:105]: This is not a really change

File: src/router/index.js [hunk 2]
```diff
@@ -92,13 +92,21 @@ export const constantRoutes = [
     path: '/documentation',
     component: Layout,
     redirect: '/documentation/index',
-    meta: { title: 'documentation', icon: 'documentation', affix: true, breadcrumb: false },
+    meta: {
```
Comment (yamelsenih) [inline src/router/index.js:95]: I don't know what is the reason for it

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/router/index.js (hunk 2 above)
Comment (EdwinBetanc0urt) [reply src/router/index.js:95]: the affix attribute controls whether it is allowed to be closed while false or not like the dashboard.

File: src/router/index.js (hunk 1 above)
Comment (EdwinBetanc0urt) [reply src/router/index.js:105]: the affix attribute controls whether it is allowed to be closed while false or not like the dashboard.

**Pull request merged**